### ISSUE TITLE
HK.6 (S:D253G) sublineage with S:L176F, S:K478T (r) with 14 seqs. Spain, UK, France, US, Portugal 

### ISSUE THREAD

**Issue by Sinickle** [cov-lineages/pango-designation#2303]
**Query:** Spike_L176F, Spike_G252V, Spike_D253G
**Oldest Sequence:** EPI_ISL_18280096, collected August 17 
**Countries circulating:** Spain (6), UK (2), US (1), France (1) 

Compared to HK.6 it has C22088T,C28657T, and does not have C22995A. 
AA mutation relative to HK.6: S:L176F, S:K478T (r)

[USHER tree](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_31f89_87be10.json?f_userOrOld=uploaded%20sample&label=id:node_3660585)
![image](https://github.com/cov-lineages/pango-designation/assets/94097102/64f0e418-3e45-4275-b731-5a7327bccdf2)

**Notes:**
HK.6 looks fairly fast at the moment. S:L176F is in enough sequences for XBB already, but doesn't necessarily look advantageous? S:K478R is one of the most convergent mutations on XBB and is advantageous, while S:K478T (r) is much more rare/not clearly advantageous. 
So overall I think it's not at all a sure thing this variant is more fit than, or even as fit as HK.6, but still reasonably interesting mutation sites.

**Comment by FedeGueli** [comment]
S:L176F is a moderate escape mutation from these ABs described by Liu here: https://www.biorxiv.org/content/10.1101/2023.04.08.536123v1.full.pdf
![Schermata 2023-10-01 alle 10 27 53](https://github.com/cov-lineages/pango-designation/assets/87669813/4f6be6bd-7a6e-4215-9d89-86f9c774473d)

**Comment by FedeGueli** [comment]
@AngieHinrichs  i do see S:T478K on Usher but it iss not thare in these sequences proposed.

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_31f89_87be10.json?c=gt-nuc_22995&gmax=23995&gmin=21995&label=id:node_3660585

**Comment by FedeGueli** [comment]
This Gisaid query catches 9 : C28657T,C22088T,A22320G,

**Comment by AngieHinrichs** [comment]
> @AngieHinrichs i do see S:T478K on Usher but it iss not thare in these sequences proposed.

Sorry, that's a symptom of overzealous branch-specific masking of reversions.  There were so many reversions of 22995 in BA.2.75, BA.5 and XBB that I figured were artifacts (and I would still bet most are artifacts), so I masked the reversion C22995A in those branches.  But unfortunately when there is a true reversion (or when a mutation is lost by recombination with a sequence that does not have it), that is now masked in the tree.  🙁
